build/charts/yorkie-monitoring/Chart.yaml

```diff
@@ -13,8 +13,8 @@ maintainers:
 
 sources:
   - https://github.com/yorkie-team/yorkie
-version: 0.4.0
-appVersion: "0.4.0"
+version: 0.4.1
+appVersion: "0.4.1"
```

Comment:
How about matching this with Yorkie version like SDK version?

Reply:
Seems like a good idea. I will reflect it.